plugins/modules/files/archive.py

```diff
@@ -43,6 +43,13 @@
       - Remote absolute path, glob, or list of paths or globs for the file or files to exclude from I(path) list and glob expansion.
     type: list
     elements: path
+  exclusion_patterns:
```

Comment:
Maybe also mention this option in `exclude_path`'s description.

Reply:
I added in a line to describe when to use `exclusion_patterns` instead, but not sure if I should add something for the intersection i.e. when `exclusion_patterns`  is used to filter out top-level files from a glob value for `path`. The primary difference is that `exclude_path` is more efficient in this scenario.

Reply:
I think it's fine to not document that, it will probably only confuse most users.